sdccc/src/main/java/com/draeger/medical/sdccc/tests/util/MdibHistorian.java

```diff
@@ -535,11 +542,26 @@ public RemoteMdibAccess applyReportOnStorage(final RemoteMdibAccess storage, fin
                     + " same version has already been applied, and is expected behavior when e.g."
                     + " descriptors update, as both a report for description and state will arrive.");
         }
-        LOG.debug(
-                "Applying report with mdib version {}, type {}",
-                ImpliedValueUtil.getReportMdibVersion(report),
-                report.getClass().getSimpleName());
-        reportProcessor.processReport(report);
+
+        if (report instanceof WaveformStream
+                || report instanceof AbstractMetricReport
+                || report instanceof AbstractAlertReport
+                || report instanceof AbstractOperationalStateReport
+                || report instanceof AbstractComponentReport
+                || report instanceof AbstractContextReport
+                || report instanceof DescriptionModificationReport) {
+            LOG.debug(
+                    "Applying report with mdib version {}, type {}",
+                    ImpliedValueUtil.getReportMdibVersion(report),
+                    report.getClass().getSimpleName());
+            reportProcessor.processReport(report);
+        } else {
+            // other reports do not modify the Mdib and hence cannot be passed into
+            //   reportProcessor.processReport().
+            // simply ignore them.
+            @SuppressWarnings("unused")
+            final var ignored = 1; // make Checkstyle happy
```

Comment:
I would suggest to just log other reports here.

Reply:
Good Idea! Done.